src/pages/general/login/components/loginForm/loginForm.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import {LoginCredentials} from '../../../../../model/login/loginCredentials';
+import {ILoginErrors} from '../../../../../model/login/loginErrors';
+import {HeaderComponent} from './components/header';
+import {FormComponent} from './components/form';
+
+interface IProps {
+  loginCredentials: LoginCredentials;
+  loginErrors: ILoginErrors;
+  updateLoginInfo: (viewModel: LoginCredentials, fieldName: string, value: string) => void;
+  loginRequest: (loginCredentials: LoginCredentials) => void;
+}
+
+export const LoginFormComponent = (props: IProps) => {
+  return (
+    <div className="container">
+      <div className="row">
+        <div className="col-md-4 col-md-offset-4">
```

Comment:
Sorry was my fault, @vicmator had these code but I delete because I think has no sense a container with hardcoded  4 width. I'll recoger the code and expose properties with bootstrap cols